scilpy/viz/color.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+
+from fury.colormap import distinguishable_colormap
+import matplotlib.pyplot as plt
+from matplotlib import colors
+import numpy as np
+
+from scilpy.viz.backends.vtk import get_color_by_name, lut_from_colors
+
+
+def convert_color_names_to_rgb(names):
+    """
+    Convert a list of VTK color names to RGB
+
+    Parameters
+    ----------
+    names : list
+        List of VTK color names.
+
+    Returns
+    -------
+    colors : list
+        List of RGB vtkColor.
+    """
+
+    return [get_color_by_name(name) for name in names]
+
+
+BASE_10_COLORS = convert_color_names_to_rgb(["Blue",
+                                             "Yellow",
+                                             "Purple",
+                                             "Green",
+                                             "Orange",
+                                             "White",
+                                             "Brown",
+                                             "Grey"])
+
+
+def generate_n_colors(n, generator=distinguishable_colormap,
+                      pick_from_base10=True, shuffle=False):
+    """
+    Generate a set of N colors (unicity not guaranteed, based on the generator)
```

Comment:
> unicity not guaranteed, based on the generator

https://pypi.org/project/distinctipy could be used ? I am joking but kind of not, but I am joking. But, you know

Reply:
I should make the docstring better. In its default parametrization, with `distinguishable_colormap`, colors are unique. But you can change the generator :wink: